src/utils/openCloseComponent.ts

```diff
@@ -8,20 +8,50 @@ export interface OpenCloseComponent {
    */
   readonly el: HTMLElement;
 
+  /**
+   * When true, the component opens.
+   */
+  open?: boolean;
+
+  /**
+   * Specifies property on which active transition is watched for.
+   */
+  activeTransitionProp: string;
```

Comment:
We should probably rename this to just `transitionProperty` or `openTransitionProperty`?